### ISSUE TITLE
[VoiceOver] Notification Settings concerns

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12911]
<img width="320" alt="iPhone Xs — 12 2 2019-11-08 14-24-48" src="https://user-images.githubusercontent.com/198826/68511601-91bea700-0233-11ea-81b7-d060b15f1206.png">

- [ ] The _View All_ does not have the `.button` trait. It should also have a hint that it expands the list.
- [ ] The site cells can be joined better by adding “Site Name at Site URL”

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
